sgpt/app.py

```diff
@@ -28,6 +30,11 @@ def main(
         show_default=False,
         help="The prompt to generate completions for.",
     ),
+    version: Optional[bool] = typer.Option(
```

Comment:
It can be implemented in more elegant way using callbacks, like we have it with couple other [functions](https://github.com/TheR1D/shell_gpt/blob/main/sgpt/app.py#L105).